python/pyplugin_installer/installer.py

```diff
@@ -118,7 +118,8 @@ def __init__(self):
                 self.uninstallPlugin(key, quiet=True)
                 updateAvailablePlugins()
                 if plugin_is_active:
-                    settings.setValue("/PythonPlugins/watchDog/" + key, True)
+                    settings.setValue("/PythonPlugins/watchDog/" + key,
```

Comment:
I think we should switch to a new key here -- I'm worried changing the value type for an existing key may cause issues if a profile is shared with older qgis versions...

Reply:
@nyalldawson Done. My intention was to share the key between versions (see the compatibility note in the description above), so one can prevent the other from crash. But you're right, that benefit would be negligible.